managed/data_source_cluster_name.go

```diff
@@ -79,46 +80,8 @@ func (r dataClusterNameType) GetSchema(_ context.Context) (tfsdk.Schema, diag.Di
 				}),
 			},
 			"backup_schedules": {
-				Computed: true,
-				Attributes: tfsdk.ListNestedAttributes(map[string]tfsdk.Attribute{
-
-					"state": {
-
-						Description: "The state of the backup schedule. Used to pause or resume the backup schedule. Valid values are ACTIVE or PAUSED.",
-						Type:        types.StringType,
-						Computed:    true,
-					},
-
-					"cron_expression": {
-						Description: "The cron expression for the backup schedule.",
-						Type:        types.StringType,
-						Computed:    true,
-					},
-
-					"time_interval_in_days": {
-						Description: "The time interval in days for the backup schedule.",
-						Type:        types.Int64Type,
-						Computed:    true,
-					},
-
-					"retention_period_in_days": {
-						Description: "The retention period of the backup schedule.",
-						Type:        types.Int64Type,
-						Computed:    true,
-					},
-
-					"backup_description": {
-						Description: "The description of the backup schedule.",
-						Type:        types.StringType,
-						Computed:    true,
-					},
-
-					"schedule_id": {
-						Description: "The ID of the backup schedule. Created automatically when the backup schedule is created. Used to get a specific backup schedule.",
-						Type:        types.StringType,
-						Computed:    true,
-					},
-				}),
+				Computed:   true,
+				Attributes: tfsdk.ListNestedAttributes(getBackupScheduleDsAttributes()),
```

Comment:
To be honest I prefer to have everything inline, and not looking accross the code.

Reply:
Yeah, I thought we need to add an attribute based on the feature flag. But, we don't want this in the TF state if the feature flag is not set as of now.